azure-functions/acbs-function/src/functions/activity-create-deal-investor.js

```diff
@@ -27,7 +26,7 @@ const handler = async (payload) => {
     }
 
     const { dealIdentifier, investor } = payload;
-
+    const mandatoryFields = ['effectiveDate', 'currency'];
```

Comment:
I think this remain at the top like other ACBS functions

Reply:
Actually just noticed these are being moved in other functions so all good I think - just want to make sure they're aligned.

Reply:
Yep good point, they all are aligned with local scope.